Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:21.7-alpine as build
+
+RUN apk add --update git nano curl python3 py3-setuptools make g++\
+   && rm -rf /var/cache/apk/*
+
+WORKDIR /app
+
+# Root monorepo package
+COPY lerna.json package*.json ./
+
+# Dev Panel
+COPY packages/yii-dev-panel/package*.json packages/yii-dev-panel/
+
+# Dev Panel SDK
+COPY packages/yii-dev-panel-sdk/package*.json packages/yii-dev-panel-sdk/
+
+# Dev Toolbar
+COPY packages/yii-dev-toolbar/package*.json packages/yii-dev-toolbar/
```

Comment:
What changes more often?

Reply:
I think yii-dev-panel itself + sdk

Reply:
```suggestion
# Dev Toolbar
COPY packages/yii-dev-toolbar/package*.json packages/yii-dev-toolbar/

# Dev Panel
COPY packages/yii-dev-panel/package*.json packages/yii-dev-panel/

# Dev Panel SDK
COPY packages/yii-dev-panel-sdk/package*.json packages/yii-dev-panel-sdk/
```

Reply:
Reason: layer with toolbar won't be rebuilt every time other layers change.

Reply:
There is copying of 2 files in 3 directories. I don't think it could have perf troubles with missing layer caching 😁